spec/views/hq/sponsors/index_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'will_paginate/array'
```

Comment:
This actually appears in the code in many places, but I think phasing it out each time it's encountered is a good idea...